draft-ietf-lsr-isis-fast-flooding.xml

```diff
@@ -361,7 +361,7 @@
 			<t>When the O-flag (Ordered acknowledgement) is set, the LSPs will be
 			acknowledged in the order they are received: a
 			PSNP acknowledging N LSPs is acknowledging the
-			N oldest LSPs received. The order inside the
+			N oldest unacknowledged LSPs. The order inside the

```

Comment:
I think the original text should be retained.
From the POV of the receiver, every LSP received is considered "unacknowledged" i.e., the receiver does not keep track as to whether the LSP was previously received and acknowledged or not. If it was received then it MUST be acknowledged. "Collapse" of acknowledgements is only possible if the receiver receives multiple copies of the same LSP before it has sent any acknowledgement.

My comment seems consistent with a point you make in your proposed email reply to John:

"Bruno] Thinking about this, I don't believe so.
The receiver is receiving LSPs. It does not matter for the receiver whether they have been acknowledged not."

Reply:
Thank you Les for your reply and for elaborating on the reasons. 
I'll revert back to the original text.
(May be an extra reason is that we don't want to imply that we are changing the way IS-IS acknowledge the LSPs. Changing the wording could imply a change. It's very unlikely that the ISO spec is refering to unacknoledged LSP.)

Reply:
Yes, that makes sense. agree